### ISSUE TITLE
ember canary / module unification, Unknown Option in replace-with-external.js

### ISSUE THREAD

**Issue by NullVoxPopuli** [BBVAEngineering/ember-route-helpers#7]
Build Error (Babel) in ember-route-helpers/helpers/replace-with-external.js

[BABEL] ember-route-helpers/helpers/replace-with-external.js: Unknown option: direct.compileModules

**Comment by josex2r** [comment]
Hi @NullVoxPopuli.
Can you create a repo to be able to reproduce the issue?

**Comment by NullVoxPopuli** [comment]
yup, https://github.com/NullVoxPopuli/emberclear

that one doesn't currently have this addon installed, cause I wanted to keep on movin.

But steps to reproduce would be:
```bash
git clone <url>
cd emberclear
rm src/components/transition-to.ts
yarn
ember install ember-route-helper
yarn start
# visit: localhost:4200/setup/completed
```

**Comment by josex2r** [comment]
It seems to be a problem with Babel.

This addon has old dependencies (including Babel `v5`). When adding it to your project there is a conflict with the version that you are using (Babel `v6`).
Upgrading this addon dependencies should fix the problem but I'm going to do some checks after making the changes.

**Comment by NullVoxPopuli** [comment]
Neato, thanks!

**Comment by josex2r** [comment]
Published `1.0.0`, this should fix the issue :)

**Comment by NullVoxPopuli** [comment]
It did! https://github.com/NullVoxPopuli/emberclear/commit/c37492bc6741cbe2d3d1986ae3b59932c6e2da98


thanks a ton!
